packages/cli/src/commands/integration/types.ts

```diff
@@ -37,6 +37,19 @@ export interface IntegrationProduct {
   metadataSchema: MetadataSchema;
 }
 
+export interface Configuration {
+  id: string;
+  integrationId: string;
+  ownerId: string;
+  slug: string;
+  teamId: string;
+  userId: string;
+  scopes: string[];
+  source: string;
+  installationType: string;
```

Comment:
`'marketplace' | ...`?

Reply:
Can add 👍 this was originally just a simplified type to have some general structure. I'll keep `| string` afterwards to protect the CLI from schema drift over time.

Reply:
Ended up pulling the `installationType` up into a type to be shared across the relevant interfaces.